### PR TITLE
5.0.1 version tick and bug fixes

### PR DESCRIPTION
**Pull request by Kelvinrr**
## Description

This should contain the bug fixes added from the previous support sprint and a version tick of 5.0.1

## Motivation and Context

Part of the 5.0.1 release 

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Documentation change (update to the documentation; no code change)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have read and agree to abide by the [Code of Conduct](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/Code-Of-Conduct.md)
- [ ] I have read the [**CONTRIBUTING**](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/CONTRIBUTING.md) document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.
- [ ] I have added myself to the [.zenodo.json](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/.zenodo.json) document.
- [ ] I have added any user impacting changes to the [CHANGELOG.md](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/CHANGELOG.md) document.

## Licensing
This project is mostly composed of free and unencumbered software released into the public domain, and we are unlikely to accept contributions that are not also released into the public domain. Somewhere near the top of each file should have these words:

> This work is free and unencumbered software released into the public domain. In jurisdictions that recognize copyright laws, the author or authors of this software dedicate any and all copyright interest in the software to the public domain.

- [ ] I dedicate any and all copyright interest in this software to the public domain. I make this dedication for the benefit of the public at large and to the detriment of my heirs and successors. I intend this dedication to be an overt act of relinquishment in perpetuity of all present and future rights to this software under copyright law.

### CODE REVIEW

**Comment by Kelvinrr** [comment]
Some notes:

* I wasn't sure whether or not to include app conversion, I did anyways with the thought of avoiding potential merge conflicts down the line 
* I wasn't sure if this is supposed to be an RC or not, I assume not?

File: CHANGELOG.md [hunk 1]
```diff
@@ -35,6 +35,20 @@ release.
 
 ## [Unreleased]
 
+### Changed
```
Comment (jessemapel) [inline CHANGELOG.md]: These need to be under a new header for 5.0.1 and there needs to be a new link at the bottom.

**Comment by jessemapel** [comment]
I agree about app conversions. If they are easy to include then go for it.

We don't do RCs for bug fix releases, they just go out the door.

File: CHANGELOG.md [hunk 2]
```diff
@@ -35,6 +35,22 @@ release.
...
+- isisVarInit.py no longer writes a "cat" statement by default to the activate scripts which cause the ISIS version information to be written on conda activate.  This can be included in those scripts via a command line option to isisVarInit.py.  Also, a quiet option is provided to isisVarInit.py to suppress its own writing to standard out, if needed.
```
Comment (jessemapel) [inline CHANGELOG.md]: This doesn't appear to be in 5.0.1, it should probably not be added to the changelog for this branch.

**Review by jessemapel — APPROVED** [review]

**Pull request merged**